src/Symfony/Component/Console/Tests/Command/CommandTest.php

```diff
@@ -439,26 +436,41 @@ public function testCommandAttribute()
         $this->assertSame(['f'], $command->getAliases());
     }
 
-    public function testAttributeOverridesProperty()
+    /**
+     * @group legacy
+     */
+    public function testCommandAttributeWithDeprecatedMethods()
     {
-        $this->assertSame('my:command', MyAnnotatedCommand::getDefaultName());
-        $this->assertSame('This is a command I wrote all by myself', MyAnnotatedCommand::getDefaultDescription());
+        $this->assertSame('|foo|f', Php8Command::getDefaultName());
+        $this->assertSame('desc', Php8Command::getDefaultDescription());
+    }
```

Comment:
Shouldn't this test and the other one below expect a deprecation message?

Reply:
They do, which is why I moved them into a new method with the legacy group, did I miss something?

Reply:
You may `use ExpectDeprecationTrait` in your test class and call `$this->expectDeprecation('...');` at the top of your test case. No need to add `symfony/deprecation-contracts` in composer.json of Console as it's already there 🙂